Gemfile

```diff
@@ -54,6 +54,10 @@ group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
```

Comment:
- [ ] It would be nice if you use includes in the posts index page you have an N+1 problem there so it would be nice if you includes in the posts index method in the posts controller 👍

```ruby
@posts = @user.posts.includes(:comments)
```

Reply:
- Hey, according to the previous reviewer, you should try to fix the n+1 problem in the post controller. you're yet to implement this. please, kindly add these missing requested changes.

Reply:
Can you be more specific about Where is that? You are pointing to the `Gemfile` here.
@zieeco

Reply:
@mahammad-mostafa, I mentioned it. read my comment well, I said in your post controller []()